spec/mailers/referee_mailer_spec.rb

```diff
@@ -1,5 +1,6 @@
 require 'rails_helper'
 
+# rubocop:disable RSpec/NestedGroups
```

Comment:
This cop seems sensible (https://github.com/rubocop-hq/rubocop-rspec/issues/148). But we're only in this position because we've got a feature flag on, and that'll go away. So :+1: to ignoring it.

Reply:
Exactly my thinking 👍